editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -284,9 +291,12 @@ private void addDescriptionAsTooltip(Parameter parameter, Label label) {
 	 * @return
 	 */
 	protected CamelModel getCamelModel(AbstractCamelModelElement modelElement) {
-		CamelModel model = modelElement.getCamelFile().getCamelModel();
-		if (model == null) {
-			return null;
+		CamelModel model = null;
+		if (modelElement != null) {
+			model = modelElement.getCamelFile().getCamelModel();
+		}
+		if (model == null && lastSelectedEP != null) {
```

Comment:
that is usually the case...sadly sometimes there is still no modelElement generated for the new subelement and in that case we need to do the check.